dvc/repo/__init__.py

```diff
@@ -60,7 +60,7 @@ class Repo(object):
     from dvc.repo.get_url import get_url
     from dvc.repo.update import update
 
-    def __init__(self, root_dir=None):
+    def __init__(self, root_dir=None, skip_checks=False):
```

Comment:
I'm really worried about this ending very badly. We should at least hide it by making it `_skip_checks` and putting a comment/docstring that people should not use it at all, unless they really know what they are doing. We need to clarify the reasoning behind it.

Reply:
How about having a internal API to disable?
```python
dvc = Repo()
dvc._skip_checks()
```

Reply:
We don't guarantee any backward compatibility here anyway.

Reply:
I am fine with other ways though.

Reply:
@skshetry Great idea! That way it is well hidden and is not hurting our eyes in the public interface.

@Suor Could you please adjust your PR to do that? E.g.
```
dvc = Repo()
dvc._skip_checks = True
```
would probably be the best. Or maybe `_disable_dag_checks`, but naming is up to you.

Reply:
Changed to `repo._skip_graph_checks = True`.